### PR TITLE
Improving the Login UI

### PR DESCRIPTION
**Pull request by luquiceno**
Using Flowbite an open source library, I improved the login UI. And I changed the structure of the code a little, adding the components folder.

### CODE REVIEW

File: src/App.css
Comment (sebasutp) [inline src/App.css:1]: Don't comment out unnecessary code, just remove it. Less code makes it easier for people to navigate and understand it. We can always bring the code back if we need to using git :)

File: src/App.jsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
-import { useState } from 'react'
-import { BrowserRouter, Routes, Route, Navigate, useNavigate } from 'react-router-dom';
+//import { useState } from 'react'
```
Comment (sebasutp) [inline src/App.jsx]: Delete

File: src/index.css [hunk 2]
```diff
@@ -1,4 +1,8 @@
-:root {
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+/* :root {
```
Comment (sebasutp) [inline src/index.css]: Delete

File: src/main.jsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+//import { StrictMode } from 'react'
```
Comment (sebasutp) [inline src/main.jsx]: Delete

**Review by sebasutp — COMMENTED** [review]
Looks good. I left some comments with the issues. In addition to that, please add a screenshot to the pull request showing how the changes look like in the frontend

File: src/App.jsx (hunk 1 above)
Comment (luquiceno) [reply src/App.jsx]: Thanks for your feedback.

**Comment by luquiceno** [comment]
This is a screenshot to see how looks until now the UI Login.

Screenshot_1

**Review by sebasutp — APPROVED** [review]

**Pull request merged**